homeassistant/components/slack/sensor.py

```diff
@@ -0,0 +1,53 @@
+"""Slack platform for sensor component."""
+from __future__ import annotations
+
+from slack import WebClient
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+import homeassistant.util.dt as dt_util
+
+from . import SlackEntity
+from .const import ATTR_SNOOZE, DOMAIN, SLACK_DATA
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Slack select."""
+    async_add_entities(
+        [
+            SlackSensorEntity(
+                hass.data[DOMAIN][entry.entry_id][SLACK_DATA],
+                SensorEntityDescription(
+                    key="do_not_disturb_until",
+                    name="Do Not Disturb Until",
```

Comment:
```suggestion
                    name="Do not disturb until",
```

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/
Entities will have their own name (without device, area). Or, they may optionally set the name to None (in that case they inherit the device name).

Then set https://developers.home-assistant.io/docs/core/entity/#entity-naming